R/explain.R

```diff
@@ -130,6 +130,10 @@ explain.default <- function(model, data = NULL, y = NULL, predict_function = NUL
     }
   } else {
     n <- nrow(data)
+    # as was requested in issue #155
+    if (is.null(rownames(data))) {
```

Comment:
does it work with tibble?

Reply:
It is impossible for tibble not to have rownames

Reply:
```
Warning message:
Setting row names on a tibble is deprecated. 
```